examples/simulator/client.mjs

```diff
@@ -58,6 +54,13 @@ let init = () => {
     });
 }
 
+let respondWithVal = (s,a,v,t) => {
```

Comment:
Yo should use `const` rather than `let`, because I don't think you are actually mutating